src/main/java/nextstep/security/authorization/manager/SecuredAuthorizationManager.java

```diff
@@ -0,0 +1,47 @@
+package nextstep.security.authorization.manager;
+
+import nextstep.security.authentication.Authentication;
+import nextstep.security.authentication.AuthenticationException;
+import nextstep.security.authorization.AuthorizationDecision;
+import nextstep.security.authorization.ForbiddenException;
+import nextstep.security.authorization.Secured;
+import org.aopalliance.intercept.MethodInvocation;
+import org.springframework.core.annotation.AnnotationUtils;
+
+import java.lang.reflect.Method;
+import java.util.Collection;
+import java.util.Set;
+
+public class SecuredAuthorizationManager implements AuthorizationManager<MethodInvocation> {
+
+    private final AuthorizationManager<Collection<String>> authorizationManager;
+
+    public SecuredAuthorizationManager() {
+        this.authorizationManager = new AuthoritiesAuthorizationManager();
+    }
+
+    @Override
+    public AuthorizationDecision check(Authentication authentication, MethodInvocation invocation) {
+        Method method = invocation.getMethod();
+
+        if (authentication == null) {
+            throw new ForbiddenException();
+        }
```

Comment:
```suggestion
        if (authentication == null) {
            return AuthorizationDecision.ACCESS_DENIED;
        }
```

권한에 따른 예외 처리는 Interceptor의 역할로 넘겨주면 어떨까요?

Reply:
예외 책임은 Interceptor에서 하는 것이 맞는 것 같아서 수정 하였습니다 !